framework/include/utils/SlepcSupport.h

```diff
@@ -80,13 +80,18 @@ void clearFreeNonlinearPowerIterations(const InputParameters & params);
 /**
  * Form matrix according to tag
  */
-void moosePetscSNESFormMatrixTag(SNES snes, Vec x, Mat mat, void * ctx, TagID tag);
+void moosePetscSNESFormMatrixTag(
+    SNES snes, Vec x, Mat mat, SparseMatrix<Number> & libmesh_mat, void * ctx, TagID tag);
 
 /**
  * Form multiple matrices for multiple tags
  */
```

Comment:
It will be nice to add more doxygen comments, like using `@param` for these functions.